doc/conf.py

```diff
@@ -124,9 +124,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '0.0.15'
+version = '0.0.16'
```

Comment:
This file has a lot of `pycodestyle` and `pylint` remarks, however this can be cleaned up in a separete issue.